Palo Alto Networks/paloalto-ngfw/ingest/parser.yml

```diff
@@ -75,6 +75,120 @@ pipeline:
           - pkts_received
         delimiter: ","
 
+  # TRAFFIC CSV with hight Resolution timestamp
```

Comment:
I think it's preferable to extend the previous stage instead to create a new one.
I believe that the `dsv.parse-dsv` is able to manage when records haven't the same number of columns defined in the columnnames property. Can you check that?